lib/bokslut/__tests__/asset-service.test.ts

```
@@ -48,17 +50,72 @@ describe('DEFAULT_ACCOUNTS_BY_CATEGORY', () => {
+        asset: '1090',
+        accumulated: '1099',
+        expense: '7810',
+      })
+    },
+  )
+
+  it('leaves 1090/1099 unflagged in the BAS chart, so the K2 gate passes them', () => {
+    expect(getBASReference('1090')?.k2_excluded).toBe(false)
+    expect(getBASReference('1099')?.k2_excluded).toBe(false)
+    // Inside the 1010-1099 window the immaterial Zod/service range checks allow.
+    expect('1090' >= '1010' && '1099' <= '1099').toBe(true)
```

Biome (2.5.6)

[error] This comparison uses the same expression on both sides.

(lint/suspicious/noSelfCompare)